packages/lib-content/src/screens/Educate/Educate.js

```diff
@@ -202,7 +201,7 @@ function Educate() {
                     // "See this <0>blog post</0> for details."
                     components={[
                       <Anchor
-                        key='group-stats-announcement'
+                        key='stats-in-classrooms-blog'
                         href='' // Need url after blog post
```

Comment:
This is an October update. See https://docs.google.com/spreadsheets/d/10k8yyZsRjaJens7BdffD6VUpoInZNLEPNQNgKSSTgpM/edit?disco=AAABVyWg6nA